### PR TITLE
Add code highlighting notes to README

### PR DESCRIPTION
**Pull request by zerolab**
Replaces #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,38 @@ that shouldn't be difficult to implement (patches welcome).
 ### Installation
 Alpha release is available on Pypi - https://pypi.org/project/wagtail-markdown/ - installable via `pip install wagtail-markdown`. It's not a production ready release.
 
+The SimpleMDE editor uses FontAwesome for its widget icons. You need to add the
```
Comment (tomdyson) [inline README.md]: would it also work with [wagtailfontawesome](https://gitlab.com/alexgleason/wagtailfontawesome)?

Comment (zerolab) [reply README.md]: ah, yes! I keep forgetting about it

Comment (zerolab) [reply README.md]: added note with link and installation instructions

**Comment by vinay87** [comment]
Hey @kevinhowbrook will you be merging this?

**Pull request merged**